Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -978,7 +978,8 @@ protected virtual void OnWaitingForPlayers()
         /// <param name="player">The <see cref="Player"/> who will be shown the message.</param>
         protected virtual void ShowPickedUpMessage(Player player)
         {
-            player.ShowHint(string.Format(Instance!.Config.PickedUpHint.Content, Name, Description), Instance.Config.PickedUpHint.Duration);
+            if (Instance!.Config.PickedUpHint.Show)
```

Comment:
Null check is not needed for Instance, but for hint

Reply:
no, everything is alright
hint cant be null
plugin is nullable